pkg/workloadapi/workload.proto

```diff
@@ -215,6 +216,33 @@ enum TunnelProtocol {
   // Future options may include things like QUIC/HTTP3, etc.
 }
 
+// NativeTunnel describes how an application or gateway
+// can consume tunnel information directly.
+message NativeTunnel {
+  // A target natively handles this type of traffic.  
+  GatewayProtocol protocol;
```

Comment:
Why is it gatewayProtocol? Sidecar can also support it

Reply:
I think john's suggestion for ApplicationProtocol is good.